projects/client/RabbitMQ.Client/src/client/api/QueueingBasicConsumer.cs

```diff
@@ -90,6 +90,7 @@ namespace RabbitMQ.Client
     /// }
     /// </code></example>
     /// </remarks>
+    [Obsolete("Deprecated. Use EventingBasicConsumer or Subscription instead")]
```

Comment:
Subscription is largely the same thing. I suspect it also doesn't support automatic recovery, so we shouldn't mention it.